node/src/service.rs

```diff
@@ -212,70 +268,186 @@ where
+        |client,
+         prometheus_registry,
+         telemetry,
+         task_manager,
+         relay_chain_node,
+         transaction_pool,
+         sync_oracle,
+         keystore,
+         force_authoring| {
+            let slot_duration = cumulus_client_consensus_aura::slot_duration(&*client)?;
+
+            let proposer_factory = sc_basic_authorship::ProposerFactory::with_proof_recording(
+                task_manager.spawn_handle(),
+                client.clone(),
+                transaction_pool,
+                prometheus_registry.clone(),
+                telemetry.clone(),
+            );
+
+            let relay_chain_backend = relay_chain_node.backend.clone();
+            let relay_chain_client = relay_chain_node.client.clone();
+            Ok(build_aura_consensus::<
+                sp_consensus_aura::sr25519::AuthorityPair,
+                _,
+                _,
+                _,
+                _,
+                _,
+                _,
+                _,
+                _,
+                _,
+            >(BuildAuraConsensusParams {
+                proposer_factory,
+                create_inherent_data_providers: move |_, (relay_parent, validation_data)| {
+                    let parachain_inherent =
+                        cumulus_primitives_parachain_inherent::ParachainInherentData::create_at_with_client(
+                            relay_parent,
+                            &relay_chain_client,
+                            &*relay_chain_backend,
+                            &validation_data,
+                            id,
+                        );
+                    async move {
+                        let time = sp_timestamp::InherentDataProvider::from_system_time();
+
+                        let slot =
+                            sp_consensus_aura::inherents::InherentDataProvider::from_timestamp_and_duration(
+                                *time,
+                                slot_duration.slot_duration(),
+                            );
+
+                        let parachain_inherent = parachain_inherent.ok_or_else(|| {
+                            Box::<dyn std::error::Error + Send + Sync>::from(
+                                "Failed to create parachain inherent",
+                            )
+                        })?;
+                        Ok((time, slot, parachain_inherent))
+                    }
+                },
+                block_import: client.clone(),
+                relay_chain_client: relay_chain_node.client.clone(),
+                relay_chain_backend: relay_chain_node.backend.clone(),
+                para_client: client.clone(),
+                backoff_authoring_blocks: Option::<()>::None,
+                sync_oracle,
+                keystore,
+                force_authoring,
+                slot_duration,
+                // We got around 500ms for proposing
+                block_proposal_slot_portion: SlotProportion::new(1f32 / 24f32),
```

Comment:
do we want these values for `block_proposal_slot_portion`  and `max_block_proposal_slot_portion`?

Reply:
This is what its set here https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.9/node/src/service.rs#L437-L439

What should be preferred values?

Reply:
@ltfschoen